docs/vercel.md

```diff
@@ -42,7 +42,7 @@ On the configuration page:
 - Override the build command to:
 
 ```bash
-cd ../.. && pnpm run build:saleor-app-checkout
+cd ../.. && pnpm run build --filter=saleor-app-checkout
```

Comment:
@mmiszy since this is something our users must do, maybe we can create some aliases in package.json:
```
"build:checkout": "pnpm run build --filter=saleor-app-checkout"
```

Reply:
this should be:

```suggestion
cd ../.. && pnpm run build --filter=saleor-app-checkout...
```

without `...` we wouldn't build dependencies of saleor-app-checkout (for example `ui-kit`) and instead use cached version 🙈

https://turborepo.org/docs/core-concepts/filtering#include-dependencies-of-matched-packages

We already had issues with Turborepo cache, it's really nasty thing to debug :/

Reply:
> @mmiszy since this is something our users must do, maybe we can create some aliases in package.json:
> 
> ```
> "build:checkout": "pnpm run build --filter=saleor-app-checkout"
> ```

Do we add a separate command for every package in the repo?

Reply:
@mmiszy I think commands for apps would be enough (storefront, saleor-app-checkout, checkout)

Reply:
Okay, I can take care of this.

Reply:
@mmiszy I only mean root level scripts that can hide turborepo details, as an extra sugar